config/config.go

```diff
@@ -335,6 +338,9 @@ func (cfg Config) Validate() error {
 			"ClientID", cfg.OIDC.ClientID,
 			"ClientSecret", cfg.OIDC.ClientSecret,
 		),
+		validateEnable("SMTP", cfg.SMTP.Enable,
+			"From", cfg.SMTP.From,
+		),
```

Comment:
Also need address to connect to set:
```suggestion
			"From", cfg.SMTP.From,
			"Address", cfg.SMTP.Address,
		),
```